### DISCUSSION TITLE
Zigbee2mqtt Mac OS docker and CC2531

### DISCUSSION THREAD

**Discussion by boesing** [Koenkk/zigbee2mqtt#5593] [General]
Hey there,

I am on MPB 2019 and when plugging in my CC2531, I can see it as `/dev/tty.usbmodem12345`.
I've tried to follow [these instructions](https://www.zigbee2mqtt.io/information/docker.html) to start zigbee2mqtt on Mac OS.
Sadly, docker reports:
```
docker: Error response from daemon: error gathering device information while adding custom device "/dev/tty.usbmodem12345": no such file or directory.
``` 

But actually, that device is available.

```bash
$ docker run  -it    -v $(pwd)/data:/app/data    --device=/dev/tty.usbmodem12345:/dev/ttyACM0    -e TZ=Europe/Berlin    -v /run/udev:/run/udev:ro    --privileged=true    koenkk/zigbee2mqtt
```

I've to add `:/dev/ttyACM0` because if I dont do that, zigbee2mqtt complains about missing `/dev/ttyACM0` when starting up.

Am I doing something wrong? Sadly, I am not that deep into docker so I'd appreciate any help.

**Comment by Koenkk** [comment]
Docker on mac doesn't support device passthrough so this won't work: https://docs.docker.com/docker-for-mac/faqs/#can-i-pass-through-a-usb-device-to-a-container

**Comment by boesing** [comment]
Mh, how did you manage to use the Stick in this video on your mac?
https://www.youtube.com/watch?v=kcRj77YGyKk

**Reply by Koenkk** [reply]
I don't use docker, just bare metal: https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html
